File: C#/C# Basics/First Steps in Coding/Exercise Tasks/Task7_FruitMarket.cs
using System;

namespace Task7_FruitMarket
{
    class Task7_FruitMarket
    {
        static void Main(string[] args)
        {
            double strawberryPrice = double.Parse(Console.ReadLine());
            double bananas = double.Parse(Console.ReadLine());
            double oranges = double.Parse(Console.ReadLine());
            double raspberries = double.Parse(Console.ReadLine());
            double strawberries = double.Parse(Console.ReadLine());


            double raspberryPrice = strawberryPrice * 0.5;
            double orangePrice = raspberryPrice - (0.40 * raspberryPrice);
            double bananaPrice = raspberryPrice - (0.80 * raspberryPrice);



            Console.WriteLine("{0:F2}",bananas * bananaPrice + strawberries * strawberryPrice + oranges * orangePrice + raspberries * raspberryPrice);
        }
    }
}

File: C++/Basics/First Steps in Coding/Exercise Tasks/Task7_FruitMarket.cpp
// Task7_FruitMarket.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>

using namespace std; 

int main()
{


    double strawberryPrice;
    cin >> strawberryPrice;

    double bananas;
    cin >> bananas;

    double oranges;
    cin >> oranges;

    double raspberries;
    cin >> raspberries;

    double strawberries; 
    cin >> strawberries;
    double allStrawberries = strawberries * strawberryPrice;

    double allRaspberries = 0.5 * strawberryPrice * raspberries;
    double allOranges = oranges * (0.5 * strawberryPrice - 0.4 * (0.5 * strawberryPrice));
    double allBanans = bananas * (0.5 * strawberryPrice - 0.8 * (0.5 * strawberryPrice));


    cout.setf(ios::fixed);
    cout.precision(2);
    cout << allBanans + allOranges + allRaspberries + allStrawberries;
}
